hal/src/electron/pinmap_hal.c

```diff
@@ -50,12 +40,7 @@
  * GPIO_PinSource15: D5, LVLOE_UC
  */
 
-
-
-/* Private typedef -----------------------------------------------------------*/
-
-STM32_Pin_Info __PIN_MAP[TOTAL_PINS] =
-{
+Hal_Pin_Info __PIN_MAP[TOTAL_PINS] = {
```

Comment:
Let's avoid using symbols starting with `_` as those are reserved for internal compiler/libc/libstdc++ stuff.

```suggestion
static Hal_Pin_info s_pin_map[TOTAL_PINS] = {
```

Same thing in other implementation files.

Reply:
Fixed